### PR TITLE
WDP200303-10

### PR DESCRIPTION
**Pull request by jurecki**
menu has been changed.
I used ul<li structure, after hover 'ul' element, user see  all categories from initState

### CODE REVIEW

**Comment by Varenthein** [comment]
Please add a proper description.

**Comment by Varenthein** [comment]
And where is the description?

**Comment by jurecki** [comment]
Zrzut ekranu 2020-03-20 o 10 25 01

File: src/components/features/ProductSearch/ProductSearch.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-      <select name='' id=''>
-        <option value=''>Select a category</option>
-      </select>
+      <FontAwesomeIcon icon={faListUl} className={styles.icon} />
+      <div className={styles.list}>
+        Select a category
+        <ul>
+          {initialState.categories.map(item => (
+            <li key={item.id}>{item.name}</li>
+          ))}
```
Comment (Varenthein) [inline src/components/features/ProductSearch/ProductSearch.js]: Text in li element should be a link :) That's it. Correct that and we're good to go.

Comment (jurecki) [reply src/components/features/ProductSearch/ProductSearch.js]: done :)

**Pull request merged**